extensions/dacpac/src/wizard/api/dacFxConfigPage.ts

```diff
@@ -156,6 +163,7 @@ export abstract class DacFxConfigPage extends BasePage {
 		)
 			.withProperties({
 				required: true,
+				validationErrorMessage: localize('dacfx.invalidFileNameErrorMessage', "Invalid file name"),
```

Comment:
I'd suggest adding more information on what specifically is invalid about the name. There's a bunch of stuff we check so it might not be clear which one is causing the failure

Reply:
Added more specific error messages